src/behat/Api/Context/ApiContext.php

```diff
@@ -504,4 +504,97 @@ function() use ($host, $service, &$hostId, &$serviceId) {
 
         return [$hostId, $serviceId];
     }
+
+    /**
+     * Send a downtime to a service
+     *
+     * @When I send a downtime on my service :service from host :host
+     */
+    public function iSendDonwtimeOnServiceFromHost(string $service, string $host)
+    {
+        [$hostId, $serviceId] = $this->iWaitUntilServiceIsMonitored($service, $host);
+
+        $objDateTime = new \DateTime('NOW');
+        $start = $objDateTime->format(\DateTime::ISO8601);
+        $objDateTime->add(new \DateInterval('PT1H'));
+        $end = $objDateTime->format(\DateTime::ISO8601);
+
+        $data = json_encode(
+            array(
+                "start_time" => $start,
+                "end_time" => $end,
+                "is_fixed" => true,
+                "duration" => 3600,
+                "author_id" => 1,
+                "comment" => "Downtime set by API",
+            )
+        );
+        $this->iSendARequestTo(
+            'POST',
+            '/beta/monitoring/hosts/' . $hostId . '/services/' . $serviceId . '/downtimes',
+            $data
+        );
+    }
+
+    /**
+     * List downtime for a service
+     *
+     * @When I list downtime of my service :servcie from host :host
+     */
+    public function iListDonwtimeOfServiceFromHost(string $service, string $host)
+    {
+        [$hostId, $serviceId] = $this->iWaitUntilServiceIsMonitored($service, $host);
+
+        $response = $this->iSendARequestTo(
+            'GET',
+            '/beta/monitoring/hosts/' . $hostId . '/services/' . $serviceId . '/downtimes'
+        );
+        $this->container->execute("php /usr/share/centreon/cron/downtimeManager.php", 'web');
```

Comment:
Why do you run downtimeManager ? You add a realtime downtime using api, isn't it ?